src/main/java/com/zemnov/salon/service/StatisticsService.java

```diff
@@ -0,0 +1,55 @@
+package com.zemnov.salon.service;
+
+import com.zemnov.salon.model.Order;
+import com.zemnov.salon.repository.OrderRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Service
+public class StatisticsService {
+
+    @Autowired
+    private OrderRepo orderRepo;
+
+    public Map<String, Integer> countedServiceTypes(){
```

Comment:
As i understand, it simple calculates how many orders belongs to each service type. Do we write our own query and use GROUP BY sql feature ?